Add blizzard style

karst opened the pull request:
This PR adds a replacement for Ice style. 

I made this song a while back with my buddy Greyfade. Asked him if it was ok to clean it up and make a loopable version of it and he was a-okay with it. 

Let me know if I need to do any changes.

karst committed:
Add blizzard style

karst committed:
Add blizzard style

Remove comma

Gymnasiast commented:
I think it might be a bit too bassy. The original were not base-heavy at all, and you’re hearing this song with all kinds of other noises mixed into it. What I fear is that the players will hear mostly bass once in game. (Also keep in mind that most users will have worse audio equipment than you.) I think the melody and treble should probably be more emphasised. And perhaps a few more snowy effects, there are quite a few at the beginning, but not so many after.

I think the basis of the song is still good. It’s okay that it’s more Jalmaan than Allister Brimble - it makes the OpenRCT2 soundtrack interesting in its own right. Just making sure it evokes the same feeling and works in practice.

Hope my rambling makes some sense. ;-)

karst committed:
Update audio file

Thisone ups the volume of the wind in the bridge and lowers the sub/bass in the master by about 6db equalizing the track to be more mid/high focussed rather than a soundscape of bass.

karst committed:
Change from Snow to Ice style replacement

tupaschoal reviewed (commented):
Personally I don't love the "vocals" (for lack of knowledge on music) that comes here and there. I think it sounds too fast for something that is snow, I'd expect something somber

karst commented:
I have heard complaints about this before. Can you check out the 122 bpm version I posted in #open-sound-and-music in discord and see if you like that better?

karst committed:
Update to 122 BPM Version

Gymnasiast commented on replacements/README.md:
```diff
@@ -42,7 +42,7 @@ Since the merry-go-round music is a playlist of fairground organ this might requ
 |                           | Roman fanfare style         | Empire style                     | Alternative |
 |                           | Summer style                | Spring style                     | Alternative |
-|                           | Snow style                  | Winter style                     | Alternative |
+|                           | Snow style                  | Winter style                     | Alternative | 
```
```suggestion
|                           | Snow style                  | Winter style                     | Alternative |
```

Gymnasiast committed:
Apply suggestions from code review
